src/ActionBar/ActionBar.test.js

```diff
@@ -74,6 +74,20 @@ describe('<ActionBar />', () => {
         expect(tree).toMatchSnapshot();
     });
 
+    describe('ActionBar Header', () => {
+        test('should allow customization of header level', () => {
+            const element = mount(
+                <ActionBar.Header
+                    description=''
+                    level={2}
```

Comment:
for this, and all of them, I think we should use `headingLevel` `headLevel`, or something similar

Reply:
I tend to agree.  I would vote for `headingLevel`.  I think if this was a `Heading` component that only emitted the `<h#>` tag, you could make a strong case for just `level`, but since this becomes a passthrough prop, I like being more explicit.